admin/data/sql/migrations/028.do.add-pupil-restart-constraints.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE [mtc_admin].[pupilRestart] WITH CHECK ADD CONSTRAINT [FK_pupilRestart_pupilRestartCode_id] FOREIGN KEY([pupilRestartCode_id])
```

Comment:
i would put the table constraints in the same migration as the table, they are atomic

Reply:
ok will refactor this then